src/Config.cpp

```diff
@@ -138,16 +140,24 @@ void OnSetConfig()
     char   *cfg    = cmdMessenger.readStringArg();
     uint8_t cfgLen = strlen(cfg);
 
-    bool maxConfigLengthNotExceeded = configLengthEEPROM + cfgLen + 1 < MEM_LEN_CONFIG;
-    if (maxConfigLengthNotExceeded) {
-        MFeeprom.write_block(MEM_OFFSET_CONFIG + configLengthEEPROM, cfg, cfgLen + 1); // save the received config string including the terminatung NULL (+1) to EEPROM
-        configLengthEEPROM += cfgLen;
-        cmdMessenger.sendCmd(kStatus, configLengthEEPROM);
-    } else
-        cmdMessenger.sendCmd(kStatus, -1); // last successfull saving block is already NULL terminated, nothing more todo
+    if (configStoredInEEPROM() || !configStoredInFlash()) {
+        bool maxConfigLengthNotExceeded = configLengthEEPROM + cfgLen + 1 < MEM_LEN_CONFIG;
+        if (maxConfigLengthNotExceeded) {
+            // save the received config string including the terminatung NULL (+1) to EEPROM
+            MFeeprom.write_block(MEM_OFFSET_CONFIG + configLengthEEPROM, cfg, cfgLen + 1);
+            configLengthEEPROM += cfgLen;
+            cmdMessenger.sendCmd(kStatus, configLengthEEPROM);
+        } else
+            // staus message to connector, failure on setting config
+            // connector does not check for status = -1
+            cmdMessenger.sendCmd(kStatus, -1);
```

Comment:
For now the connector reports a successfull upload of the config, even if `-1` is send back.
If no message is send back, the connector shows an error message (due to time out) that the config uploads fails.

Does it make sense to uncomment `cmdMessenger.sendCmd(kStatus, -1);` as long as the connector does not handle this?
There were discussion before to implement a `kError` message which would be sensefull also in this case ;)